scheduler/src/cook/mesos/api.clj

```diff
@@ -740,6 +766,13 @@
                   "success" "failed")
                 :job.state/running "running"
                 :job.state/waiting "waiting")
+        constraints (->> job
+                         :job/constraint
+                         (map util/remove-datomic-namespacing)
+                         (map (fn [{:keys [attribute operator pattern]}]
+                                (->> [attribute (clojure.string/upper-case (name (util/without-ns operator))) pattern]
+                                     (remove nil?)
```

Comment:
I was using the same set up as marathon's constraints, https://mesosphere.github.io/marathon/docs/constraints.html which have an optional pattern, but looking more closely they blend job and group constraints in a way that I'm not sure we want. I will make it required.